introduction.md

```diff
@@ -0,0 +1,2 @@
+We found your "introduction.md" file, but it looks to be empty, or just too short. We really want to know something about you! 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.